### ISSUE TITLE
Questions and Feedback

### ISSUE THREAD

**Issue by smoore4moma** [WaltersArtMuseum/api-thewalters-org#66]
[Comment]

Below are comments and questions....only Contributors can tag "issues" otherwise I would have tagged it as a Question.  

First, I love the Sandbox feature.  That is really cool.

I recently fired up Visual Studio 2013 and set up up a SPA with Web API with the intent of making a public API for MoMA (where I work) using TMS as the backend database.  The API is almost done and then I noticed that you have done almost the exact same thing (see image).  Clearly I will need a different Bootstrap theme ;O)

So, here are the questions that I am sure you must have dealt with....

1) I am using OAuth only.  You're not using it at all.  I am wondering what your thoughts are about that.
2) I am trying to have two routes that return a single object, using either ObjectID or ObjectNumber.  Something like this:

GET api/objects/{object_id:int}
GET api/objects/{object_number}

So basically if you provide an integer it will look for ObjectID, otherwise it will use ObjectNumber (e.g., 123.2013).  I noticed that someone already mentioned this.  I don't want to put it in the QueryString because it is not really REST for this particular service.  I have spent 3+ hours trying to get this to work.  Have you found a way?  I ended up adding an "action" to make it separate routes.

Again, thanks for your advice and great work!

Steve MOORE
MoMA DBA

PS, Happy Thanksgiving!  I work remotely from France so I end up writing comments on GitHub when everyone else is drinking beer and watching football!

![img](https://cloud.githubusercontent.com/assets/4989910/5218995/db6097d2-7620-11e4-9672-3abf2a6f69ed.jpg)
